fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -432,12 +432,16 @@ public LogicalPlan visitInsertIntoQuery(InsertIntoQueryContext ctx) {
                 colNames,
                 ImmutableList.of(),
                 partitions,
-                ConnectContext.get().getSessionVariable().isEnableUniqueKeyPartialUpdate(),
+                false,
```

Comment:
this change is not about hint, please explain why do this change. add some comment if necessary

Reply:
Here it uses the session variable in the parser to analyze weather to use the semantic of partial update for the current insert statement.  However, we can't apply the effect of the hint of the current insert statement before we parse the statement itself, so I move the analysis of the semantic of partial update of the current insert statement to `BindSink`